src/azurelinux/3.0/cross/arm-net9.0/Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:azurelinux-3.0-crossdeps-builder-net9.0-local AS builder
 ARG ROOTFS_DIR=/crossrootfs/arm
 
-RUN /scripts/eng/common/cross/build-rootfs.sh arm xenial --skipunmount
+# The arm rootfs targets Ubuntu 22.04, which is the first version with a
+# glibc that supports 64-bit time_t. See https://github.com/dotnet/core/discussions/9285.
+RUN /scripts/eng/common/cross/build-rootfs.sh arm jammy no-lldb --skipunmount
```

Comment:
IIRC, this supports 64-bit time_t, but there's `#define` needed to actually use it. Do we need to set that somewhere?